### PR TITLE
azure government module - closes #260

### LINKED ISSUE #260: understand Azure Government services

### PR DESCRIPTION
**Pull request by BenHesketh21**

### CODE REVIEW

File: topics/azure/modules/government-services/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [Exercises](#exercises)
+
+<!--TOC_END-->
+## Overview
+
+Azure Government is a cloud platform that is separate to normal Azure that has an extra focus on `security`, `compliance` and `transparency`.
+
+## Azure Government
+
+US government agencies or any partners of the US government have access to Azure Government which provide world class security and protection of their resources to help comply with government standards.
```
Comment (LukeBenson) [inline topics/azure/modules/government-services/README.md]:
```suggestion
US government agencies or any partners of the US government have access to Azure Government which provide world class security and protection for their resources to help comply with government standards.
```

File: topics/azure/modules/government-services/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Azure Government use isolated data centers and networks that are only located in the US.
+
+## Azure Government vs global Azure
+
+Both the cloud providers use the same underlying technologies.
```
Comment (LukeBenson) [inline topics/azure/modules/government-services/README.md]:
```suggestion
Both cloud providers use the same underlying technologies.
```
or 'both of the cloud...' it doesn't matter which

File: topics/azure/modules/government-services/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+However, there are some differences, the main one being that the isolated data centers are in the US only and only screened users are given access to operate on the sovereign cloud.
```
Comment (LukeBenson) [inline topics/azure/modules/government-services/README.md]: You talk about the sovereign cloud here without ever explaining what it is

File: topics/azure/modules/government-services/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [Azure Government](#azure-government)
+- [Azure Government vs global Azure](#azure-government-vs-global-azure)
+- [Other sovereign clouds](#other-sovereign-clouds)
+- [Tutorial](#tutorial)
+- [Exercises](#exercises)
+
+<!--TOC_END-->
+## Overview
+
+Azure Government is a cloud platform that is separate to normal Azure that has an extra focus on `security`, `compliance` and `transparency`.
```
Comment (BenHesketh21) [inline topics/azure/modules/government-services/README.md]:
```suggestion
Azure Government is a cloud platform that is separate to normal Azure that has an extra focus on `security`, `compliance` and `transparency`. This is known as a sovereign cloud.
```

**Pull request merged**